x-pack/plugins/fleet/public/hooks/use_request/agents.ts

```diff
@@ -350,3 +350,17 @@ export function sendGetAgentsAvailableVersions() {
     version: API_VERSIONS.public.v1,
   });
 }
+
+export function sendGetAgentStatusRuntimeField() {
+  return sendRequest<string>({
```

Comment:
It will be better to `sendRequestForRq` that throw on error instead of returning the error like `sendRequest` does

Reply:
updated